python-recipes/recommendation-systems/collaborative_filtering_schema.yaml

```diff
@@ -0,0 +1,40 @@
+index:
+    name: movies
+    prefix: movie
+    storage_type: json
+
+fields:
+    - name: movieId
```

Comment:
don't necessarily need to fix but I don't like it when someThings are camel and other_things are snake

Reply:
I don't really like it either. It's to match the data in the csv files which mix both.

Reply:
Heard. Catch22. Ship it or .rename but that's a bit of an annoying refactor haha